group/short.go

```diff
@@ -226,38 +233,34 @@ func (s *wScl) UnmarshalBinary(b []byte) error {
 	copy(s.k[l-len(b):l], b)
 	return nil
 }
-
-type wHash struct {
-	sswu3mod4
-	elt FieldHasher
-	scl FieldHasher
-}
-
 func (g wG) HashToElement(b, dst []byte) Element {
 	var u [2]big.Int
-	h := g.getHasher(dst)
-	h.elt.HashToField(u[:], b)
-	Q0 := h.Map(&u[0])
-	Q1 := h.Map(&u[1])
+	g.precomputeHashParams()
```

Comment:
Should I be concerned about concurrent access to one of these?

Reply:
yes, this is not thread-safe. I fixed that now.